plugin/pick.vim

```diff
@@ -68,3 +72,16 @@ function! s:BufferListCommand()
   let buffers = map(bufnrs, 'bufname(v:val)')
   return 'echo "' . join(buffers, "\n") . '"'
 endfunction
+
+function! s:TagCommand()
+  let l:tag_file = ""
+
+  for tfile in split(&tags, ",")
+    if filereadable(expand(tfile)) == 1
+      let l:tag_file = tfile
+      break
+    endif
+  endfor
+
+  return "awk -F$'\t' '{print $1}' " . l:tag_file . "  | sort -u | grep -v '^!'"
```

Comment:
Good idea, `awk` takes it's arguments separated by spaces. So we can select the files that are readable from `&tags` and then join them by spaces when we pass the list to `awk`

Done in e798eaf